sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -383,6 +380,7 @@ MarginCalculationLauncher::computeGlobalMargin(const boost::shared_ptr<LoadIncre
       else
         maxVariation = newVariation;  // at least, one crash
     } else {
+      loadIncreaseResult.clearScenariosResults();
```

Comment:
Why adding this? Seems that it is out of the scope of the ticket?

Reply:
If the load increase of 0% fails, we step out of the MarginCalculationLauncher::launch() method and MarginCalculationLauncher::results_ contains only the LoadIncreaseResult of 0%. At this point of the code, the LoadIncreaseResult of 0% contains the SimulationResult of the load increase and of the scenarios while no scenarios was actually simulated. This code deletes the dummy scenarios to not add them in bestResults and worstResults maps in MarginCalculationLauncher::createOutputs().

Reply:
In this case we should simply add a test on the laodincrease status when collecting the result?